### PR TITLE
I changed tx modal reverted sentence

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/ProfilePage/TransferLoadingModal.tsx [hunk 1]
```diff
@@ -6,9 +6,10 @@ import { TxSTATUS } from "../../hooks/useERC721Transfer"
 
 interface Props {
   status?: string
+  reverted?: boolean
```
Comment (sweetmantech) [inline components/ProfilePage/TransferLoadingModal.tsx]: change `reverted` to `removing` ✏️

Comment (sweetmantech) [reply components/ProfilePage/TransferLoadingModal.tsx]:
```suggestion
  removing?: boolean
```

**Review by SweetmanTech — APPROVED** [review]
`reverted` => `removing` ✏️

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**